services/search/pkg/query/kql/normalize.go

```diff
@@ -0,0 +1,120 @@
+package kql
+
+import (
+	"github.com/owncloud/ocis/v2/services/search/pkg/query/ast"
+)
+
+var implicitOperatorNodeSource = "implicitly operator"
+var operatorNodeAnd = ast.OperatorNode{Base: &ast.Base{Loc: &ast.Location{Source: &implicitOperatorNodeSource}}, Value: BoolAND}
+var operatorNodeOr = ast.OperatorNode{Base: &ast.Base{Loc: &ast.Location{Source: &implicitOperatorNodeSource}}, Value: BoolOR}
+
+// NormalizeNodes Populate the implicit logical operators in the ast
+//
+// https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference#constructing-free-text-queries-using-kql
+// If there are multiple free-text expressions without any operators in between them, the query behavior is the same as using the AND operator.
+// "John Smith" "Jane Smith"
+// This functionally is the same as using the OR Boolean operator, as follows:
+// "John Smith" AND "Jane Smith"
```

Comment:
Misleading. Either a typo in line `16` (`OR`) or a typo in line `17` (`AND`)

Reply:
GOSH, what a review, you read that comment 🥇 THX!